portafly/src/i18n/locales/en/audience/accounts/listing.yml

```diff
@@ -0,0 +1,108 @@
+ admin_header: Admin
+ admin_header_description: Column listing administrators
+ signup_header: Signup date
+ signup_header_description: Column listing signup dates
+ applications_header: Applications
+ applications_header_description: Column listing applications linked to accounts, on a 1:1 relationship
+ state_header: State
+ state_header_description: Column listing the state of the accounts
+ #-------# actions_header_plural: available in shared.yml
+ actions_column_options:
+  approve: Approve
+  approve_description: Button link to approve an account
+  activate: Activate
+  activate_description: Button link to perform the activation of an account
+  act_as: Act as
+  act_as_description: Button link enabled for administrators to impersonate an account
+
+# Modal: Send email
+send_email:
+  page: Send email to admins of selected accounts
+  page_description: Body title of contextual window for sending email to administrators of the selected accounts
+  to_label: To
+  to_label_description: Header for email recipients
+  subject_label: Subject
+  subject_label_description: Header for email subject
+  message_label: Message
+  message_label_description: Header to email message
+  #-------# Send button: available in shared.yml
+  #-------# Cancel button: available in shared.yml
+  toasts:
+   send_email_start: Sending email
+   send_email_start_description: Alert message when email is being sent
+   send_email_success: Your message was sent.
+   send_email_success_description: Alert message when email is sent
+   send_email_error: Your message was not sent because of a network error. Please contact your administrator.
+   send_email_error_description: Alert message when email could not be sent
+
+# Modal: Change state
+change_state:
+  page: Change state of the selected accounts
+  page_description: Body title of contextual window for changing state of the selected accounts
+  selectedaccounts_label: Selected accounts
+  selectedaccounts_label_description: Header to list all the selected accounts
+  selectstate_label: Select state
+  selectstate_label_description: Header to request the selection of the target state
+  admonition_label: Changing the state of a developer account does not imply changing the state of its applications.
+  admonition_label_description: Admonition text to explain that the change of state of a developer account doesn't mean changing the state of any of its applications.
+  changestate_button: Change state
+  changestate_button_description: Label for a button to change the state of the selected accounts
+  #-------# Cancel button: available in shared.yml
+  toasts:
+    change_state_start: Changing accounts state
+    change_state_start_description: Alert message when state is being changed
+    change_state_success: Account state changed
+    change_state_success_description: Alert message when state is changed
+    change_state_error: Account state was not changed because of an error. Please try again.
```

Comment:
@porueesq should `change_state_success` and `change_state_error` be plural?

Reply:
Yes, they can. I'll add the strings.